<launch>
  <node pkg="orbslam2_ros" type="orbslam2_ros_node" name="orbslam2_ros_node" output="screen">
    <rosparam command="load" file="$(find orbslam2_ros)/params/param_V1_01_easy_example.yaml"/>
    <remap from="camera/left/image_raw" to="/cam0/image_raw"/>
    <remap from="camera/right/image_raw" to="/cam1/image_raw"/>
  </node>

  <node pkg="rviz" type="rviz" name="rviz"
  args="-d $(find orbslam2_ros)/launch/V1_01_easy.rviz"/>

</launch>
